travel-2018.html

```diff
@@ -0,0 +1,236 @@
+
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
+        <div class="pycon-travel">
+        	<p><u><a href="https://wikitravel.org/en/Hyderabad">Here is a handy Wiki page</a></u> describing about the various ways you can travel to Hyderabad. The main points have been highlighted below.</p>
+       		<ul>
+       			<li>
+              <h3>Via Air</h3>
+       				<p><b>To Hyderabad: </b>Domestic connectivity to Hyderabad is excellent with Indian airlines operating from here including Air India, Air India Express, Indian Airlines, Indigo Airlines, Jet Airways, JetLite, Trujet and SpiceJet. (Kingfisher is now defunct, and Trujet is new).</p>
+
+       				<p><b>From the Airport to the Venue: </b>Once you arrive at Hyderabad airport, one option is to take the air-conditioned buses run by the airport (Pushpak) to Hi-Tec City (Opposite Shilparamam) at prices varying between Rs 150 and Rs. 250 based on the distance of travel. The buses have a frequency of a bus every 30 min from 3:30AM-11PM and every hour at midnight, 1AM, 2AM and 3AM and travel time varies from 45 mins to 2 hrs depending on time of the day and traffic conditions. You can reach the designated points and then take an auto or metered cab from there.</p>
+
+					    <p>To get directly near the venue, you will have to take a bus to <b>Kothagouda</b>. Below are the timings for the buses:</p>
+
+					    <img src="img/airport_to_venue.png" alt="image">
+
+					    <p>Alternatively, you can hire metered air-conditioned radio cabs starting from Rs 40 for the first 2 km. Skycabs and Meru are approved by the airport @ 21 Rs/km and are available just after exiting the terminal building. For the rest, you need to call and book with a lead time of 15 minutes to 1 hour. These cabs charge 25% surcharge in the night (ie, Rs 26.25 per km between 2300 hours and 0500 hours). Hyderabad traffic police counter is on the ground floor with prepaid taxis.</p>
+
+					    <p><b>Beware of taxi soliciting tours at the airport greeting area; they might try to scam you into exorbitant rates (especially so in case of non- locals)</b>. The Hyderabad Traffic Police has partnered with the Airport authorities to run a counter for prepaid taxis. This is a safe option with all taxis being registered with the police. The police also runs a SHE cab service now, especially for women travellers, with female drivers and safety equipment such as a GPS connected to the Police Control Room.</p>
+
+					    <p>One can also hire a cab from many app- based services such as Uber or Ola.
+					    <u><a href="https://book.olacabs.com/?pickup_name=Rajiv%20Gandhi%20International%20Airport%20(RGIA)%2C%20Shamshabad%20Hyderabad%20Telangana&lat=17.24066&lng=78.428747&drop_lat=17.4622103&drop_lng=78.3568492&drop_name=Kondapur%2C%20Hyderabad%20Telangana%20India">Here is a fair estimate using OLA.</a></u></p>
```

Comment:
I think adding a link to booking an Ola is not needed, Hence can be removed.
#suggestion